pom.xml

```diff
@@ -83,6 +83,8 @@
         <carbon.config.version.range>[2.1.5, 3.0.0)</carbon.config.version.range>
         <carbon.securevault.version>5.0.15</carbon.securevault.version>
 
+        <carbon.analytics.version>3.0.36</carbon.analytics.version>
```

Comment:
Bump the carbon-analytics version to the latest

Reply:
Resolve in a4fdf3a6c9273b94df3533d31c73ce818529c09f